client/role/role.controller.js

```diff
@@ -0,0 +1,78 @@
+/* global angular */
+/* global _ */
+/* jshint node: true */
+'use strict';
+
+angular.module('app.role')
+  .controller('RoleCtrl', ['$scope', '$location', 'userService','alertService', 'user', 'adminLevels', 'locations', 'facilities', 'programs', 'facilityPrograms',  function($scope, $location, userService, alertService, user, adminLevels, locations, facilities, programs, facilityPrograms){
+
+    $scope.user =  user;
+    $scope.userTypes =  ['Dashboard', 'Mobile'];
```

Comment:
This should not be hard coded... consider placing in a config / settings file, could also be gotten from the parent application api in some cases.